api/src/main/java/org/openmrs/module/fhir2/providers/r3/EncounterFhirResourceProvider.java

```diff
@@ -158,4 +161,21 @@ public IBundleProvider searchEncounter(@OptionalParam(name = Encounter.SP_DATE)
 		    participantReference, subjectReference, encounterType, id, lastUpdated, includes, revIncludes));
 	}
 	
+	/**
+	 * The $everything operation fetches all the information related the specified encounter
```

Comment:
we could actually have this as: The $everything operation fetches all the information related to the specified encounter

Reply:
will update this.